.gitlab-ci.yml

```diff
@@ -42,23 +42,26 @@ test:
 
 test:modules-artifact-gen:
   stage: test
-  image: ubuntu:22.04
+  image: ubuntu:18.04
```

Comment:
I assume that this is because of OpenSSL, so please write that in the commit message.

Reply:
python3.9 apparently this is the only way. [fixed](https://github.com/mendersoftware/mender/pull/1316/commits/c8988152f7deb4730c35b6aeea6a7b23f9691897).

Reply:
Aha! It's good that you added it then, since I misunderstood it!

Reply:
I never fully understood it, so we are even :)